server/routes/v1/version.ts

```diff
@@ -99,3 +99,40 @@ export const resetVersionApprovals = [
     return res.json(version)
   },
 ]
+
+export const updateLastViewed = [
+  ensureUserRole('user'),
+  async (req: Request, res: Response) => {
+    const { id, role } = req.params
+    const { user } = req
+    if (!user) {
+      throw Forbidden({ code: 'user_unauthorised' }, 'Not user details found in request.')
+    }
+    const version = await findVersionById(req.user!, id, { populate: true })
+    if (!version) {
+      throw BadReq({ code: 'version_not_found' }, 'Unabled to find requested version')
+    }
+    if (user?.id !== version?.metadata.contacts[role]) {
+      throw Forbidden({ code: 'user_unauthorised' }, 'User is not authorised to do this operation.')
+    }
+    if (role === 'manager') {
+      updateManagerLastViewed(id)
+      req.log.info(
+        { code: 'version_last_viewed_updated', version: id, role },
```

Comment:
Do we need to update the logging translations with the new logs you've added in this PR?

Reply:
I would say so, yes. I think we need to make a note of any new log codes somewhere so that we can update logging, but not sure where's best.

Reply:
Do we create a confluence page to track them? Or we could create a ticket for each code that needs a new translation?